conans/client/build/compiler_flags.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+from conans.tools import unix_path
+
+
+def rpath_flags(os_build, compiler, lib_paths):
+    if not os_build:
+        return []
+    if compiler in ("clang", "apple-clang", "gcc"):
+        rpath_separator = "," if os_build == "Macos" else "="
```

Comment:
I see rpath_flags are os_build or os (one of em).
so probably better to use `if os_build in ["Macos", "iOS", "watchOS", "tvOS"]`